includes/Admin.php

```diff
@@ -91,11 +91,12 @@ public function admin_body_class( $class ) {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param string $content Default post content.
+	 * @param string   $content Default post content.
+	 * @param \WP_Post $post Post object.
```

Comment:
nit: can we align the params here? I thought PHPCS does enforce that.

Reply:
Seems like it is a known issue. https://github.com/squizlabs/PHP_CodeSniffer/issues/2379

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/4849/commits/14c2a0e58d51b5c4da99a12e55594f484a509536